### ISSUE TITLE
Why logits is the feature?

### ISSUE THREAD

**Issue by Amitayus** [tomrunia/TF_FeatureExtraction#8]
Thank you for your sharing. In your Usage introduction, Logits layer of Inception-v4 is exacted as feature using example_feat_extract.py. I wonder that  why the logits is the feature be extracted?

**Comment by alexandercameronh** [comment]
@Amitayus I believe that the feature vector that you call to be extracted depends on the ultimate usage of the feature vector. In this case, the Logits feature vector is effectively a probability distribution among the 1000 possible objects that can be extracted from the image

@tomrunia How would one extract a feature vector from somewhere earlier in the network? For example, the final convolutional layer?

**Comment by tomrunia** [comment]
@alexandercameronh You can just set the command line parameter `--layer_names` where the layer names are comma separated. Of course the layer names need to correspond to your network architecture. Your can use the feature extractor's method `print_network_summary()` to find out the layer names.

**Comment by Amitayus** [comment]
@alexandercameronh To my understand, if the Logits output is regarded as feature vector, then there is no need for the existing of classifier. Feature vector should be independent of the specific classification tasks.

**Comment by bkwwangjie** [comment]
@tomrunia Hello，I want to use the eigenvector of the image in conjunction with the text, which layers should I extract?

**Comment by alexandercameronh** [comment]
@bkwwangjie did you figure out the answer to this? If so, which layer(s) did you extract?

**Comment by alexandercameronh** [comment]
@bkwwangjie did you figure out the answer to this? If so, which layer(s) did you extract?

**Comment by Amitayus** [comment]
@alexandercameronh I remove the fully connected layer (regrading as classifier) and extract vector of `end_points['PreLogitsFlatten']` as feature of the input.

**Comment by zengsn** [comment]
@Amitayus Could please show the code?
